backend/service/k8s/pods.go

```diff
@@ -165,25 +167,28 @@ func (s *svc) checkExpectedObjectMetaFields(expectedObjectMetaFields *k8sapiv1.E
 }
 
 func podDescription(k8spod *corev1.Pod, cluster string) *k8sapiv1.Pod {
-	// TODO: There's a mismatch between the serialization of the timestamp here and what's expected
-	// on the frontend.
-	// var launch *timestamp.Timestamp
-	// if converted, err := ptypes.TimestampProto(k8spod.Status.StartTime.Time); err == nil {
-	// 	launch = converted
-	// }
+	var startTimestamp *timestamppb.Timestamp
+	startTime := k8spod.Status.StartTime
+	if startTime != nil {
+		startTimeProto, err := ptypes.TimestampProto(startTime.Time)
```

Comment:
ptypes is deprecated, use `timestamppb.New(startTime.Time)`

Reply:
Done